gnosisAgent/README.md

```diff
@@ -0,0 +1,32 @@
+## Gnosis Safe admin changes
```

Comment:
Please use the following sections in the Readme that are the standard template
- Title (you already have this one)
- Description (equivalent to Overview)
- Supported Chains
- Alerts
- Tests Data